src/models/room.js

```diff
@@ -228,6 +228,20 @@ function pendingEventsKey(roomId) {
 
 utils.inherits(Room, EventEmitter);
 
+Room.prototype.lazyDecryptEvents = async function() {
```

Comment:
Perhaps just call it `decryptEvents`...? There's nothing inherently "lazy" about the function body that I can see, it's more about the timing of when it is called, right?

```suggestion
Room.prototype.decryptEvents = async function() {
```